src/metpy/calc/thermo.py

```diff
@@ -407,6 +407,8 @@ def lfc(pressure, temperature, dewpt, parcel_temperature_profile=None, dewpt_sta
     parcel_profile
 
     """
+    _, temperature = _remove_nans(pressure, temperature)
+    pressure, dewpt = _remove_nans(pressure, dewpt)
```

Comment:
Should you be able to do this now:
```python
pressure, temperature, dewpt = _remove_nans(pressure, temperature, dewpt)
```
?

Reply:
I'll apply that to this function and the rest of the thermo calc that I applied this to.